CONTRIBUTING.md

```diff
@@ -59,6 +59,14 @@ There are also some integration tests, which can be run from the Debug pane, or
 
     npm run integrationTest
 
+#### Common Issues
+
+- 12 February 2020
+  - Consecutive test runs on are currently impacted on VS Code v.1.42.0 due to caching issues. We have opened an [issue with the VS Code team](https://github.com/microsoft/vscode/issues/90484) and we are looking into alternate fixes on our end.
```

Comment:
```suggestion
  - Consecutive test runs are currently impacted on VS Code v.1.42.0 due to caching issues. See [VS Code issue #90484](https://github.com/microsoft/vscode/issues/90484).
```

- `- 12 February 2020` can be omitted, since the issue link and use of the word "currently" signals time-sensitivity already. 
- "we are looking into..." can be omitted since it's assumed (doesn't add new information).

Reply:
I like having the date present in case we forget to remove it. I'll go ahead and add it to the issue description so we can have less nesting.